registry-manager-service-impl/pom.xml

```diff
@@ -48,6 +48,17 @@
             <artifactId>mapstruct-jdk8</artifactId>
```

Comment:
could you be so kind and update mapstruct version to the latest one

Reply:
Done